components/MenuTrigger/MenuTrigger.styled.js

```diff
@@ -0,0 +1,23 @@
+import styled from "styled-components";
+
+export const TriggerContainer = styled.div`
+  position: fixed;
+  justify-content: center;
+  align-items: end;
+  padding-bottom: 28px;
+  bottom: 0;
+  display: flex;
+  width: 100vw;
+  height: 60px;
+`;
+
+export const MenuTriggerButton = styled.button`
+  background-color: ${({ showMenu }) => (showMenu ? "transparent" : "white")};
+  border: ${({ showMenu }) => (showMenu ? "1px solid white" : "none")};
+  color: ${({ showMenu }) => (showMenu ? "white" : "black")};
```

Comment:
This is an optional suggestion, but it would increase the readability of your code. you can define the default values first and then use a single callback to conditionally style the showMenu option: 
```js
  background-color: white;
  border:  none;
  color: black;
  
  ${({showMenu}) => showMenu && css`
    background-color: transparent;
    border:  1px solid white;
    color: white;
  `}
  ```

Reply:
so true. Thanks for that!